### DISCUSSION TITLE
Compiling Stellarium in KDE Neon 6

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3690] [General]
I updated my laptop os KDE Neon and it turned to KDE Neon 6. I installed Qt-6 packages which are in in instructions. Then cmake and then make. It stopped to errors. in Kubuntu compiling went first time ok and produred executable. So, you can see what components are missing. Or needs updating of compiling instructions.

[kaannastellarium.txt](https://github.com/Stellarium/stellarium/files/14749348/kaannastellarium.txt)

**Comment by 10110111** [comment]
Try looking for the package that provides `lconvert` with `apt-file search lconvert` (may need to install `apt-file` first).

**Reply by 10110111** [reply]
Seems like [`qt6-l10n-tools`](https://packages.ubuntu.com/jammy/localization/qt6-l10n-tools) is the package.
